code/modules/mob/living/carbon/human/human.dm

```diff
@@ -859,6 +860,17 @@
 	dna.check_integrity(src)
 	return
 
+// For blinking
+/mob/living/carbon/human/proc/blink_eyes()
+	if(in_stasis || isskrell(src) || isvaurca(src) || is_diona(src) || isipc(src) || (status_flags == GODMODE))

```

Comment:
there should be a better way to do this

Reply:
Yeah, there is. I am just gonna leave it this for now. I tried using species lists, doesn't work in a better way at all. Especially given that IPCs are different species between each frame.